### PR TITLE
External links refactor 

### PR DESCRIPTION
**Pull request by mrgarris0n**
Fixes: #7 

Added missing links to external_links.yml file
Added some guidelines to yml
Created logical grouping and alphabetical order in yml
Eliminated hardwired URLs from admin guide
Fixed some missing id-s

### CODE REVIEW

File: doc/_admin-guide/200_About/README.md [hunk 1]
```diff
@@ -12,13 +12,13 @@ description: >-
...
 For sales and other inquiries, such as licensing, support, and renewals,
-visit <https://www.oneidentity.com/company/contact-us.aspx>.
+visit Contact Us page on our website.
 
 ## Technical support resources
 
 Technical support is available to One Identity customers with a valid
 maintenance contract and customers who have trial versions. You can
-access the Support Portal at <https://support.oneidentity.com/>.
+access the Support Portal [[here|oi-support-serv]].
```
Comment (HofiOne) [inline doc/_admin-guide/200_About/README.md:21]: to eliminate here maybe we could use:


Technical support for One Identity customers with a valid maintenance contract, as well as for those using a trial version, is available via our [[Support Portal|oi-support-serv]].

**Review by HofiOne — CHANGES_REQUESTED** [review]

File: doc/_admin-guide/190_The_syslog-ng_manual_pages/010_syslog-ng_conf.md [hunk 2]
```diff
@@ -251,19 +249,19 @@ listed below. For details, see the
 |amqp()|           Publishes messages using the AMQP (Advanced Message Queuing Protocol).
```
Comment (HofiOne) [inline doc/_admin-guide/190_The_syslog-ng_manual_pages/010_syslog-ng_conf.md:249]: i was wondering if it broke anything adding | to the pattern separators to find title occurrences in these table declarations as well

**Review by HofiOne — CHANGES_REQUESTED** [review]

File: doc/_admin-guide/020_The_concepts_of_syslog-ng/007_The_structure_of_a_log_message/001_IETF_syslog_messages.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ title: IETF-syslog messages
 id: adm-struct-ietf
 description: >-
     This section describes the format of a syslog message, according to the
-    [IETF-syslog protocol](https://tools.ietf.org/html/rfc5424). 
+    IETF-syslog protocol.
```
[the remark this answers is not in the document]
Comment (mrgarris0n) [reply doc/_admin-guide/020_The_concepts_of_syslog-ng/007_The_structure_of_a_log_message/001_IETF_syslog_messages.md:6]: "IETF-syslog" is a more inclusive variant than "IETF-syslog protocol", but I'll handle this in a separate PR, with link aliases.

File: doc/_admin-guide/060_Sources/025_Arr_logs/README.md [hunk 4]
```diff
@@ -29,4 +29,4 @@ The logging module is stored in the &lt;prefix&gt;&lt;module&gt; name-value pair
 
 The prefix can be modified with the `prefix()` option.
 
-This driver is a reusable configuration snippet. For details on using or writing such configuration snippets, see Reusing configuration blocks. The source of this configuration snippet can be accessed on [GitHub](https://github.com/syslog-ng/syslog-ng/blob/master/scl/arr/arr.conf).
+This driver is a reusable configuration snippet. For details on using or writing such configuration snippets, see Reusing configuration blocks. The source of the Arr configuration snippet can be accessed on GitHub.
```
[the remark this answers is not in the document]
Comment (mrgarris0n) [reply doc/_admin-guide/060_Sources/025_Arr_logs/README.md:32]: This is only for the config file links.

File: _includes/doc/admin-guide/options/gRPC-keep-alive.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 ## keep-alive()
 
-This option configures the forwarding of [gRPC keepalive pings](https://grpc.io/docs/guides/keepalive/) in syslog-ng OSE.
+This option configures the forwarding of gRPC keepalive pings in syslog-ng OSE.
```
[the remark this answers is not in the document]
Comment (mrgarris0n) [reply _includes/doc/admin-guide/options/gRPC-keep-alive.md:3]: Will create a link alias for that in a separate PR

File: doc/_admin-guide/120_Parser/021_Checkpoint_parser.md [hunk 6]
```diff
@@ -21,7 +21,7 @@ Splunk format:
 >time=1557767758|hostname=r80test|product=Firewall|layer_name=Network|layer_uuid=c0264a80-1832-4fce-8a90-d0849dc4ba33|match_id=1|parent_rule=0|rule_action=Accept|rule_uid=4420bdc0-19f3-4a3e-8954-03b742cd3aee|action=Accept|ifdir=inbound|ifname=eth0|logid=0|loguid={0x5cd9a64e,0x0,0x5060a8c0,0xc0000001}|origin=192.168.96.80|originsicname=cn\=cp_mgmt,o\=r80test..ymydp2|sequencenum=1|time=1557767758|version=5|dst=192.168.96.80|inzone=Internal|outzone=Local|proto=6|s_port=63945|service=443|service_id=https|src=192.168.96.27|
 
 If you find a message that the checkpoint-parser() cannot properly
-parse, [contact Support](https://www.syslog-ng.com/support/), so we can
+parse, contact Support, so we can
 improve the parser.
```
[the remark this answers is not in the document]
Comment (mrgarris0n) [reply doc/_admin-guide/120_Parser/021_Checkpoint_parser.md:25]: #56

**Review by HofiOne — APPROVED** [review]

**Pull request merged**